.changeset/fair-bats-destroy.md

```diff
@@ -0,0 +1,7 @@
+---
+'project-watchtower': major
+---
+
+Switched to react-helmet-async for page metadata which does not give lifecycle warnings in React 16+.
+
+If you use `WatchtowerBrowserRouter` you just need to update your imports from `react-helmet` to `react-helmet-async` and you are done. Otherwise you will need to wrap your client application in a `<HelmetProvider>`.
```

Comment:
Is this something that should live in the documentation?

Reply:
Yeah, I am working on a PR to break it up into a few different libraries, will sort readme's and docs then.

The WatchtowerBrowserRouter currently isn't documented so there wasn't anywhere to add to

Reply:
Will do in #36